### PR TITLE
added Flux.jl

### PR DESCRIPTION
**Pull request by ParthPant**

### CODE REVIEW

**Comment by ParthPant** [comment]
I hope this PR fixes the git issues of the last pr

File: data.json [hunk 1]
```diff
@@ -294,6 +294,82 @@
...
+    "kind": "snippet",
+    "package": "Flux",
+    "tags": [
+      "layers",
+      "neural-networks"
+    ]
+  },
+  {
+    "command": "model = Chain(Dense(10, 5, σ), Dense(5,3), softmax)",
+    "description": "Chains different layers to form a deep neural network model",
```
Comment (abelsiqueira) [inline data.json:321]:
```suggestion
    "description": "Chain different layers to form a deep neural network model",
```

File: data.json [hunk 2]
```diff
@@ -294,6 +294,82 @@
...
+    "kind": "snippet",
+    "package": "Flux",
+    "tags": [
+      "layers",
+      "neural-networks"
+    ]
+  },
+  {
+    "command": "opt = Descent(0.1)",
+    "description": "Creates a Gradient Descent optimiser with learning rate 0.1",
```
Comment (abelsiqueira) [inline data.json:331]:
```suggestion
    "description": "Create a Gradient Descent optimiser with learning rate 0.1",
```

File: data.json [hunk 3]
```diff
@@ -294,6 +294,82 @@
...
+    "package": "Flux",
+    "tags": [
+      "gradient-descent",
+      "neural-network",
+      "optimization"
+    ]
+  },
+  {
+    "command": "ps = Flux.params(m)",
+    "description": "Extracts the parameters of a model `m` which can be later passed on to the `gradient` function",
```
Comment (abelsiqueira) [inline data.json:342]:
```suggestion
    "description": "Extract the parameters of a model `m` which can be later passed on to the `gradient` function",
```

File: data.json [hunk 4]
```diff
@@ -294,6 +294,82 @@
...
+  },
+  {
+    "command": "df(x) = gradient(x -> 3x^2+ 2x + 1, x)[1]",
+    "description": "The gradient function takes another Julia function f and a set of arguments, and returns the gradient with respect to each argument.",
+    "kind": "snippet",
+    "package": "Flux",
+    "tags": [
+      "gradients"
+    ]
+  },
```
Comment (abelsiqueira) [inline data.json:359]:
```suggestion
```
This is actually from Zygote.

File: data.json [hunk 5]
```diff
@@ -294,6 +294,82 @@
...
+  {
+    "command": "train!(loss, params, data, opt; cb)",
+    "description": "For each datapoint `d` in `data`, compute the gradient of `loss` with respect to `params` through backpropagation and call the optimizer `opt`. An optional callback is given with the keyword argument `cb`",
```
Comment (abelsiqueira) [inline data.json:362]:
```suggestion
    "description": "Train the model. For each datapoint in `data`, compute the gradient of `loss` with respect to `params` through backpropagation and call the optimizer `opt`. An optional callback is given with the keyword argument `cb`",
```

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Thanks! I've made a few changes, see if you approve them.

File: data.json (hunk 4 above)
Comment (ParthPant) [reply data.json]: I've changed the package for this to Zygote and added a pkg description for the same

**Comment by ParthPant** [comment]
@abelsiqueira is this fine or there are some more changes?

File: data.json [hunk 6]
```diff
@@ -521,5 +588,23 @@
...
+    "description": "Automatic Differentiation in Julia",
+    "kind": "header",
+    "package": "Zygote",
+    "tags": [
+      "gradients"
+    ]
+  },
+  {
+    "command": "df(x) = gradient(x -> 3x^2+ 2x + 1, x)[1]",
+    "description": "The gradient function takes another Julia function f and a set of arguments, and returns the gradient with respect to each argument.",
```
Comment (abelsiqueira) [inline data.json:603]:
```suggestion
    "command": "gradient(f, args...)",
    "description": "Computes the gradient of f at each argument, returning a tuple.",
```
I think this is simpler and more informative of the return type.

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Hi, sorry for the delay. I made a simple small change. You can accept it directly on GitHub clicking on the commit suggestion button, if you agree with the change.

File: data.json (hunk 6 above)
Comment (ParthPant) [reply data.json:603]: This seem like a better description!

**Comment by abelsiqueira** [comment]
Thanks!

**Comment by abelsiqueira** [comment]
@all-contributors please add @ParthPant for data

**Pull request merged**